opencti-platform/opencti-front/lang/back/fr.json

```diff
@@ -192,6 +192,7 @@
   "Description": "Description",
   "Descriptions": "Descriptions",
   "Detection": "Détection",
+  "Digest period": "Période de digestion",
```

Comment:
this is weird : a digest is more like "un résumé/condensé"

Reply:
en effet :D